xml/ha_rear.xml

```diff
@@ -445,10 +445,11 @@
   </para>
 
   <para>
-   A default &rear; configuration for reference is available at
-   <filename>/usr/share/rear/conf/default.conf</filename>. Other example
-   configurations (<filename>*example.conf</filename>) are available in the
-   <filename>example</filename> subdirectory. 
+   All &rear; configuration variables and their default values are set in
+   <filename>/usr/share/rear/conf/default.conf</filename>.
+   Some example files (<filename>*example.conf</filename>) for user configurations
+   (for example, what is set in <filename>/etc/rear/local.conf</filename>)
+   are available in the <filename>examples</filename> subdirectory.
    Find more information in the &rear; man page.
   </para>
 
```

Comment:
Regarding item (3) in bsc#1040371 you may add here
a new paragraph with a text like:

You should start with a matching example config file as template
and adapt it as needed to create your particular configuration file.
You may need to pick out this and that from several example config files
and combine the pieces into your specific /etc/rear/local.conf file
that matches your particular system.
You cannot use an example config file 'as is'.
The example configs are primarily meant as hints what
config variables are of interest regarding a particular setup.